src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Extensions/DesignTimeDirectiveTargetExtension.cs

```diff
@@ -54,10 +54,7 @@ private void WriteDesignTimeDirectiveToken(CodeRenderingContext context, DesignT
         }
 
         // Wrap the directive token in a lambda to isolate variable names.
-        context.CodeWriter
-            .Write("((")
-            .Write(typeof(Action).FullName)
-            .Write(")(");
```

Comment:
💭 I think I might prefer the form on the left, and just change `((` to `((global::`